sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/SynchronousMessageSubscriberTest.java

```diff
@@ -64,15 +65,23 @@
     private SynchronousMessageSubscriber syncSubscriber;
     private AutoCloseable mocksCloseable;
 
+    private Sinks.Empty<Void> completeEmitter1;
+    private Sinks.Empty<Void> completeEmitter2;
+
     @BeforeEach
     public void setup() {
         mocksCloseable = MockitoAnnotations.openMocks(this);
 
+
         when(work1.getId()).thenReturn(WORK_ID);
         when(work1.getNumberOfEvents()).thenReturn(NUMBER_OF_WORK_ITEMS);
+        completeEmitter1 = Sinks.empty();
+        when(work1.getCompleteEmitter()).thenReturn(completeEmitter1);
```

Comment:
Is `completeEmitter1` `null`? How does `getCompleteEmitter()) == null` work in `SynchronousMessageSubscriber`?

What is the purpose of these mock?

Reply:
Actually when we use new a instance of `SynchronousReceiveWork`, the `getCompleteEmitter()` won't be null. 
Because `work1` is a mock instance:   
```
@Mock
private SynchronousReceiveWork work1;
```
So I need mock  `getCompleteEmitter()` return `completeEmitter1` to avoid nullpointer Exception when use this method in `SynchronousMessageSubscriber`